Improve documentation for partial stack setups (#974)
WordPress/openverse: documentation/guides/quickstart.md

# Quickstart guide

<!-- the main entrypoint of the Openverse documentation system -->

This guide covers the steps to get the Openverse stack running locally on your
computer. This guide is for setting up the full stack, which includes the API,
the ingestion server and the frontend.

## Stack-specific quickstarts

It is very unlikely that you want to contribute to everything, everywhere, all

> at once. In all likelihood, you intend to contribute to a narrower slice of the

Reviewer: > everything, everywhere, all at once

🙄 😆 👍

stack. In such cases, you might find it more beneficial to go through one of
these stack-specific quickstart guides.

> - [API](./api/quickstart.md)
> - [Ingestion server](./ingestion_server/quickstart.md)
> - [Frontend](./frontend/quickstart.md)
> - [Documentation](./documentation/quickstart.md)

Reviewer: ```suggestion
- [API](./api/quickstart.md)
- [Frontend](./frontend/quickstart.md)
- [Ingestion server](./ingestion_server/quickstart.md)
- [Documentation](./documentation/quickstart.md)
```

Since you've _written_ the ingestion server guide, I have no problem keeping it. But perhaps let's change the order to prioritize the frontend.

> That said, there is something very appealing about running the end-to-end of the
> stack locally, which this guide is all about.

Reviewer: ```suggestion
That said, there is something very appealing about running the full stack locally, which this guide is all about.
```

Small nit to simplify language.
